src/test/java/com/uid2/optout/vertx/OptOutServiceVerticleTest.java

```diff
@@ -127,8 +127,12 @@ public void writeIdsParallel_expect200(TestContext context) {
 
     // optout/add forwards request to remote optout/write api endpoints
     @Test
+    public void replicateWithoutOptoutRole_expect401(TestContext context) {
+        verifyStatus(context, replicateQuery(234), 401);
+    }
+    @Test
     public void replicate_expect200(TestContext context) {
-        verifyStatus(context, replicateQuery(234), 200);
+        verifyStatus(context, replicateQuery(234), 200, "Bearer test-optout-operator-key");
```

Comment:
very nitpick: can just make com.uid2.optout.vertx.TestOperatorKeyProvider#TEST_OPTOUT_KEY public can get the name directly instead of hardcoding it again to be inconsistent. Can probably say the same about com.uid2.optout.vertx.OptOutServiceVerticleTest#internalTestKey definition

Reply:
Thanks, I think this make sense, I've upated the tests